src/components/slotMeasurement/SlotMeasurements.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { SlotMeasurementRequest } from "../../types/sdk";
+import { Row } from "react-table";
+import DataTable from "../DataTable";
+import FormikInput from "../forms/Input";
+
+type SlotMeasurementProps = {
+  slotMeasurements: SlotMeasurementRequest[];
+  measurementName: string;
+  stepIncrement:string ,
+  onChangeMeasurement: (address: string, value: string) => void;
+  validateValue?: (value: string) => void;
+};
+
+/**
+ * Component to display SlotMeasurements as a table with two columns - slot address & measurement value
+ *
+ * @param slotMeasurements - SlotMeasurement data
+ * @param measurementName  - Name of the measurement
+ * @param stepIncrement - Measurement value increment step
+ * @param onChangeMeasurement - Callback for measurementValue
+ * @param validateValue - Validation to run when measurement value changes
+ *
+ */
+
+const SlotMeasurements = ({
+  slotMeasurements,
+  measurementName,
+  stepIncrement ,
+  onChangeMeasurement,
+  validateValue,
+
+}: SlotMeasurementProps) => {
+  const columns = React.useMemo(() => {
+    return [
+      {
+        Header: "Address",
+        id: "address",
+        accessor: (measurement: SlotMeasurementRequest) => measurement.address,
+      },
+      {
+        Header: `${measurementName}`,
```

Comment:
I don't think this needs to be a template literal.

Reply:
Yes, will change it.